scripts/idr0062_prediction.py

```diff
@@ -112,53 +111,53 @@ def load_labels_as_masks(conn, image):
 # Load-Model
 def load_model():
     """
-    Load an existing model from StarDist
+    Load an existing model from Cellpose
     """
-    return StarDist2D.from_pretrained('2D_demo')
+    return models.Cellpose(gpu=False, model_type='cyto')
 
 
 # Predict and save the shapes as geojson.
-def predict(data, model):
+def predict(image_id, data, model, size_z):
     """
     Predict object probabilities and star-convex polygon distances
     Convert the generated labels into geojson
     """
 
-    axis_norm = (0,1)
-    c = 1
-    img = normalize(data[0, c, :, :, :], 1,99.8, axis=axis_norm)
-    results = []
+    channels = [[0, 1]]
+    t = 0
     shapes = []
-    for i in range(len(img)):
-        new_labels, details = model.predict_instances(img[i])
-        # Convert into Polygon and add to Geometry Collection
-        for obj_id, region in enumerate(details['coord']):
-            coordinates = []
-            x = region[1]
-            y = region[0]
-            for j in range(len(x)):
-                coordinates.append((float(x[j]), float(y[j])))
-            # append the first coordinate to close the polygon
-            coordinates.append(coordinates[0])
-            shape = Polygon(coordinates)
-            properties = {
-                "stroke-width": 1,
-                "z": i,
-                "c": c,
-            }
-            shapes.append(Feature(geometry=shape, properties=properties))    
-        results.append(new_labels)
-
-    label_slices = numpy.array(results)
+    for z in range(size_z):
+        cellpose_masks, flows, styles, diams = model.eval(data[t, :, z, :, :], diameter=None, channels=channels)
+        outlines = utils.outlines_list(cellpose_masks)
+        name = str(image_id) + "_" + str(z)
+        io.outlines_to_text(name, outlines)
+        with open(name + "_cp_outlines.txt", "r") as text_file:
+            for line in text_file:
+                points = re.sub(r',([^,]*),', r',\1, ', line)
+                splited = re.split('\s+', points)
+                coordinates = []
+                for value in splited:
+                    point = re.split(',', value)
+                    if len(point) > 1:
+                        coordinates.append((float(point[0]), float(point[1])))
+                # append the first coordinate to close the polygon
+                coordinates.append(coordinates[0])
+                shape = Polygon(coordinates)
+                properties = {
+                    "stroke-width": 1,
+                    "z": z,
+                }
+                shapes.append(Feature(geometry=shape, properties=properties))
+
     gc = FeatureCollection(shapes)
-    return label_slices, gc
```

Comment:
``label_slices`` not defined now - error later in the notebook

Reply:
sorry. I forgot to save. I will sort it out